specs/predeploys.md

```diff
@@ -231,17 +231,26 @@ the L1 portion of the fee. This fee pays for using L1 as a data availability
 layer and should be added to the L2 portion of the fee, which pays for
 execution, to compute the total transaction fee.
 
-The values used to compute the L2 portion of the fee are:
+The values used to compute the L1 portion of the fee prior to the Ecotone upgrade are:
 
 - scalar
 - overhead
 - decimals
```

Comment:
TODO: need to clarify that these revert after the ecotone upgrade.
The `decimals` value is hardcoded, but I think it should also revert, since we don't use it anymore.

Reply:
tracked here now: https://github.com/ethereum-optimism/protocol-quest/issues/73